packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -14,6 +14,7 @@ export const createPlaceholderRow = (
   invoiceId: string,
   itemId = ''
 ): DraftOutboundLine => ({
+  __typename: 'InvoiceLineNode',
   availableNumberOfPacks: 0,
```

Comment:
Not ideal! I suppose a downside of using codegen and enforcing `__typename` through the code gen config